LOPART/LOPARTFunctions.py

```diff
@@ -0,0 +1,99 @@
+import LOPARTInterface
+import numpy as np
+import pandas as pd
+
+
+def runLOPART(data, labels, penalty, n_updates=-1, inf=False, penalty_unlabeled=-1):
+    if not n_updates > 0:
```

Comment:
if n_updates < 1: would be easier to understand I think

also n_updates is mostly for visualization / understanding / explanation of the algorithm, in practice you should always use n_updates=len(data)